playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -156,6 +156,7 @@ EDXAPP_ZENDESK_USER: ""
 EDXAPP_ZENDESK_URL: ""
 EDXAPP_ZENDESK_API_KEY: ""
 EDXAPP_ZENDESK_CUSTOM_FIELDS: {}
+EDXAPP_CELERYBEAT_SCHEDULER: 'djcelery.schedulers.DatabaseScheduler'
```

Comment:
Note for version upgrade:

This can be set via secure vars ([example|https://github.com/edx-olive/olive-secure/search?q=celerybeat]), so it should be fine to drop this change when rebasing Ginkgo customizations on release branch for, e.g., Koa.

Default value of this setting is `celery.beat:PersistentScheduler` on upstream [master](https://github.com/edx/configuration/blob/master/playbooks/roles/edxapp/defaults/main.yml#L176).